tmt/plans/main.fmf

```diff
@@ -98,10 +98,10 @@ prepare:
     script: |
       export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
       export OS=$(uname | awk '{print tolower($0)}')
-      export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.36.0
+      export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.37.0
       curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
       chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
-      operator-sdk olm install
+      operator-sdk olm install --version v0.28.0
```

Comment:
Is this forcing an older or newer version? Why do we need to control the version at all?

Reply:
It is forcing of older version, since new version 0.29.0 cannot be installed on kind due to some sec context requirements. This is a hotfix to have running env and I will investigate more sec context on kind.